### ISSUE TITLE
Image explaining the use of alt texts has no alt text

### ISSUE THREAD

**Issue by kvnloughead** [mdn/content#12713]
This image [here](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML) of the Mozilla red dinosaur illustrating how alt texts appear over images on hover has no alt text. 

![image](https://user-images.githubusercontent.com/37159177/152055022-bb9e638b-4cab-4074-ac3b-5a5c0d46dceb.png)

Suggested alt text:

> Alt text tooltip reading "The Mozilla red dinosaur" hovering over the Mozilla red dinosaur.

**Comment by deepug9787** [comment]
Hi, I have a question about this issue.

If you look at the code example under the "[Text alternatives](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML#text_alternatives)" section, there are four images - the first one has a source attribute, the second one has both source and alt attributes, the third one has source, alt, and title attributes and the last one has an aria-labelledby attribute.

The image mentioned by @[kvnloughead](https://github.com/kvnloughead) as part of this issue refers to the third image in that code example whereas the sentence "In this case, we are not using the alt attribute at all" mentioned by @[teoli2003](https://github.com/teoli2003) in his [comment](https://github.com/mdn/content/pull/12767#issuecomment-1030775128) here refers to the fourth image. This can also be confirmed by the actual name of the image: title-attribute.png.

So I think this is a pertinent change. Can I go ahead and fix this issue by adding the alt and title attributes for the image?
